### DISCUSSION TITLE
Welcome to megaels Discussions!

### DISCUSSION THREAD

**Discussion by kachurovskiy** [kachurovskiy/megaels#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gerritv** [comment]
Hi, I'm Gerrit. Presently in last stages of building an ELS V2 from original chipmaker,ru source code (then found this implementation which confirms that it will work!). Using it on a King KC1022ML, like a Grizzly G0602.
Steppers are belt drive to lead screws. X axis is 1:1, Z is 1:2 for more torque.
Everything is wired but having problems with my TB6560 drivers, missing steps. I think I know the solution, have to get some 100pf chip capacitors to replace the wrong ones on the board. I have a box for setup controls and one for the joystick, limit buttons and handwheel.

**Reply by kachurovskiy** [reply]
Have you considered getting some quality drivers instead e.g. Rtelligent R60?

**Reply by gerritv** [reply]
This is what I have, until I see if this whole thing is worth it there is no more $ going into it :-) The 100pf caps are known to solve the issue on the red 4 axis boards from a bunch of years ago, I just never bothered before this.

**Comment by pantonvich** [comment]
Hello - started my rels more then a year ago - but got side tracked until a month ago and restarted it. I think I’ve got all my wiring glitches fixed and so how I’m working on the software configuration… I’m putting this on an eBay mini lathe 14 inch I bought years ago. My x and z interfaces and spindle holder are 3D printed from a mashup of other peoples designs and mine. I’m happy to share. I’m in atlanta ga usa
Pete

**Comment by ic569** [comment]
Hi all, searching the net for an ELS I stumble upon Maxim's MegaELS youtube video and github pages. I downloaded zip file with the code but have trouble as it doesn't compile giving the following error message:
![Screenshot 2023-01-13 184236](https://user-images.githubusercontent.com/122629910/212373091-b2f9e63e-a3dd-4967-9416-6407f8173a8d.png)
any thoughts?

Thanks,
Konstantinos

**Reply by kachurovskiy** [reply]
Hi @ic569, make sure you download the whole repo using e.g. https://github.com/kachurovskiy/megaels/archive/refs/heads/main.zip so that after you unzip it, not only the main file (`Digital_Feed_7e2.ino`) is present but also all the files in the same folder (e.g. `Sphere.ino`)

**Reply by ic569** [reply]
Hi Maxim, issue solved when I moved unziped file folder in Arduino IDE projects folder.
Thanks for your prompt reply!!!

**Comment by Carlos-Gobel** [comment]
Hello,

I am currently implementing the project that you made available in your repository. However, over the last few weeks, the link of the Cone_Info and Thread_Info tables has become unavailable, and I am no longer able to perform the necessary calculations.

Broken link: http://kachurovskiy.com/megaels/Digital_Feed_7e2/tables.html

I was wondering if it would be possible for you to upload the tables to a new link or make them available in some other way?

Your project has been incredibly helpful, and I would like to thank you for developing it.

Thank you in advance.

**Reply by kachurovskiy** [reply]
Thanks for letting me know, fixed that page now. New link - https://kachurovskiy.github.io/megaels/Digital_Feed_7e2/tables.html
